common/source/docs/common-blheli32-passthru.rst

```diff
@@ -48,7 +53,7 @@ The Pass-Through feature allows BLHeli32, AM32, and BLHeli_S ESCs to be upgraded
    ArduPilot firmware supports the pass-through protocol with up-to-date BLHeli32, AM32 firmware and BLHeliSuite32, or BLHeli_S firmware and BLHeliSuite only.
 
 .. warning::
-   For pass-through to function, the autopilot must be configured to use one of the DShot protocols.  If you wish to eventually use one of the other protocols (e.g. PWM, OneShot125) that the ESC supports, you may still configure the ESCs using Pass-Through (e.g. change motor directions, set min/max values, etc) but then finally re-configure the autopilot to *not* use DShot.  Once the autopilot and ESCs are rebooted the ESC should auto-detect that the ESCs are no longer using DShot.
+   For pass-through to function, the autopilot must be configured to use one of the DShot protocols using :ref:`SERVO_BLH_OTYPE <SERVO_BLH_OTYPE>` even if you use PWM.  If you wish to eventually use one of the other protocols (e.g. PWM, OneShot125) that the ESC supports, you may still configure the ESCs using Pass-Through (e.g. change motor directions, set min/max values, etc) but then finally re-configure the autopilot to *not* use DShot.  Once the autopilot and ESCs are rebooted the ESC should auto-detect that the ESCs are no longer using DShot.
```

Comment:
This is not correct. Setting MOT_PWM_TYPE or Q_M_PWM_TYPE is also fine, but if you set SERVO_BLH_MASK then you must set SERVO_BLH_OTYPE

Reply:
Perhaps this should link to the bdshot setup where it can explain these cases more in detail?

Reply:
```suggestion
   For pass-through to function, the autopilot must be configured to use one of the DShot protocols.  If you wish to eventually use one of the other protocols (e.g. PWM, OneShot125) that the ESC supports, you may still configure the ESCs using Pass-Through (e.g. change motor directions, set min/max values, etc) but then finally re-configure the autopilot to *not* use DShot.  Once the autopilot and ESCs are rebooted the ESC should auto-detect that the ESCs are no longer using DShot.
```